fabric-events-lifecycle-v0/src/main/resources/fabric-events-lifecycle-v0.mixins.json

```diff
@@ -4,6 +4,7 @@
   "compatibilityLevel": "JAVA_8",
   "mixins": [
     "MixinMinecraftServer",
+    "MixinPlayerManager",
```

Comment:
You will probably need to port this to lifecycle-events-v1 when I get it through (it is very close to last call)